packages/visualizations/src/components/Map/types.ts

```diff
@@ -1,16 +1,22 @@
-import type { Feature, FeatureCollection, Position } from 'geojson';
+import type { Feature, FeatureCollection, Position, BBox } from 'geojson';
 import type { FillLayerSpecification, Popup } from 'maplibre-gl';
-import type { Color, ColorsScale } from '../types';
+import type { DebouncedFunc } from 'lodash';
+import type { ColorScales, Color } from '../types';
 
 export interface ChoroplethOptions {
-    shapes: ChoroplethShapeValue;
-    colorsScale?: ColorsScale;
+    shapes: ChoroplethShapeValues;
+    colorsScale?: ColorScales;
     legend?: MapLegend;
     aspectRatio: number;
     activeShapes?: string[];
     interactive?: boolean;
-    emptyValueColor: Color;
-    tooltip: { label: ChoroplethTooltipFormatter };
+    emptyValueColor?: Color;
+    tooltip?: {
+        labelFormatter?: ChoroplethTooltipFormatter;
+        labelMatcher?: ChoroplethTooltipMatchers;
+    };
+    bbox?: BBox | undefined;
+    filter?: string[] | number[] | undefined;
```

Comment:
Yes this interface looks good to me 👍

> I suggest we implement the bare minimum we need for now

I agree. And it won't be difficult to allow array of values in the future if needed.

Reply:
Agreed too, but maybe `filter?: { [key: string]: string }`. Or is it harder to manipulate afterward ?

Reply:
I finally kept support for arrays, because otherwise I'd have to rewrite all tests to use départements instead of régions 😬 It's 10 characters more in the code so I think it's alright